.github/workflows/janitor-clusters.yaml

```diff
@@ -41,3 +40,12 @@ jobs:
         working-directory: ./scripts
         run: ./janitor-clusters.sh ${{ matrix.project }}
         shell: bash
+
+      # Notify on failures
```

Comment:
This, and the `set -euo pipefail` in the script, prevents us from missing this in the future.